helper_funcs.py

```diff
@@ -729,39 +787,82 @@ def plotFactMask(rev, LSD):
             rfm = file.mask[:].copy()
 
     # Make the master figure
-    fig = plt.figure(layout="constrained", figsize=(15, 10))
+    fig = plt.figure(layout="constrained", figsize=(18, 15))
     axis = fig.subplots(1, 1)
 
-    # Plot the factorized mask
-    im = axis.imshow(
-        mask,
-        extent=(0, 360, 400, 800),
-        cmap="binary",
-        aspect="auto",
-        interpolation="nearest",
-    )
+    patches = []
+
     # Overlay the full mask if it exists
     if "rfm" in locals():
+        # Include fully flagged regions
+        rfm |= _mask_flags(file.time, LSD)[np.newaxis]
+        # Trim the padded time regions
+        sel = _select_CSD_bounds(file.time, LSD)
+        rfm = rfm[:, sel]
+
+        cmap = matplotlib.colormaps["Reds"]
```

Comment:
Recommend colourmap change to make it easier to discern between factorizable mask (black) and daily mask (currently red)